docs/compendium/public_share.md

```diff
@@ -15,7 +15,7 @@ Upon successful download from the public share, the `id` for the new compendium
 curl -d "content_type=compendium_v1" \
     -d "share_url=https://uni-muenster.sciebo.de/index.php/s/G8vxQ1h50V4HpuA"  \
     --cookie "connect.sid=<code string here>" \
-     http://…:8088/api/v2/compendium
+     http://…:8088/api/v1/compendium
```

Comment:
Actually, the port should be left our here, as the API spec is for the whole API, not for one microservice. Please update :-) (and other URLs, too).